### ISSUE TITLE
PHP 7 compatibility

### ISSUE THREAD

**Issue by WouterJ** [sulu-io/sulu#2118]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 73fceeb
| Browser Version | -

#### Actual Behavior

Fatal error is triggered: `Fatal error: Cannot use 'String' as class name as it is reserved in .../sulu/vendor/doctrine/phpcr-odm/lib/Doctrine/ODM/PHPCR/Mapping/Annotations/String.php" on line 20`

#### Expected Behavior

I would assume it to just work, as https://github.com/sulu-io/sulu/pull/1906 indicated that sulu supports PHP 7 and composer.json tells me `>=5.5` is supported (even PHP 8 btw!).

#### Steps to Reproduce

```
$ php --version
PHP 7.0.x ...
$ git clone https://github.com//sulu-io/sulu
$ composer install
```

#### Possible Solutions

You have to use Doctrine PHPCR-ODM 1.3 and use the `@Field` annotations. Also, make sure to remove the registration of the `annotations.php` file, annotations can be autoloaded using the Composer autoloader.

**Comment by wachterjohannes** [comment]
we have updated all dependencies in current develop to be PHP7 compatible. this will be released in 1.2 end of march.

**Comment by patkar** [comment]
@WouterJ I can't find your given commit sha in the repo?

Btw. the bundle tests (their `bootstrap.php`) are broken because of this error.
